packages/teams-js/src/public/meeting.ts

```diff
@@ -560,4 +591,78 @@ export namespace meeting {
       sendMessageToParent('meeting.appShareButton.setOptions', [shareInformation]);
     }
   }
+
+  /**
+   * Ext App will handle the Audio(Mic & Speaker). Teams to go Audio less (No Audio)
```

Comment:
I'm not sure how to interpret this sentence. I think it's supposed to describe what this function does, and if so it should be more explicitly described. Something like "Have the external application handle audio (mic & speaker) and turn off host audio". It would also be good to describe when host audio will resume